packages/rancher-monitoring/rancher-monitoring/generated-changes/patch/templates/thanos-ruler/servicemonitor.yaml.patch

```diff
@@ -1,18 +1,10 @@
 --- charts-original/templates/thanos-ruler/servicemonitor.yaml
 +++ charts/templates/thanos-ruler/servicemonitor.yaml
-@@ -38,10 +38,21 @@
-     tlsConfig: {{ toYaml .Values.thanosRuler.serviceMonitor.tlsConfig | nindent 6 }}
+@@ -41,6 +41,16 @@
+     {{- if .Values.thanosRuler.serviceMonitor.metricRelabelings }}
+     metricRelabelings: {{- tpl (toYaml .Values.thanosRuler.serviceMonitor.metricRelabelings | nindent 6) . }}
      {{- end }}
-     path: "{{ trimSuffix "/" .Values.thanosRuler.thanosRulerSpec.routePrefix }}/metrics"
--{{- if .Values.thanosRuler.serviceMonitor.metricRelabelings }}
-+
-     metricRelabelings:
--{{ tpl (toYaml .Values.thanosRuler.serviceMonitor.metricRelabelings | indent 6) . }}
--{{- end }}
-+    {{- if .Values.thanosRuler.serviceMonitor.metricRelabelings }}
-+    {{ tpl (toYaml .Values.thanosRuler.serviceMonitor.metricRelabelings | indent 6) . }}
-+    {{- end }}
```

Comment:
metric relabellings should still be included here as well

Reply:
Same as above: https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/thanos-ruler/servicemonitor.yaml#L41-L43